### ISSUE TITLE
Data Manager class

### ISSUE THREAD

**Issue by prparikh02** [prparikh02/LeapMotion-Sign-Language-Interpretor#8]
Design and implement a data manager class that stores relevant frame data in an object with metadata. For example, a DataManager class that can be initialized by the frames array, a label array, etc. and has getter functions.

\# Ex:
class FrameManager(self, data, labels, ...)

**Comment by kiennguyen94** [comment]
To interface with Keras library, the DataManager class should have a generator function that yields a tuple of (data, label). Ex:

```python
def generate_arrays_from_file(path):
    while 1:
    f = open(path)
    for line in f:
        # create numpy arrays of input data
        # and labels, from each line in the file
        x1, x2, y = process_line(line)
        yield ({'input_1': x1, 'input_2': x2}, {'output': y})
    f.close()

```

Please refer to [Keras documentation](https://keras.io/models/model/), `fit_generator` section for more detail.
